### DISCUSSION TITLE
Add Rich Text Editor for milestone description on GH Projects

### DISCUSSION THREAD

**Discussion by fariassdev** [community/community#35894] [Repositories]
Hi, we are now trying GH Projects and we miss the Rich Text Editor on the milestone description.

Currently, it supports markdown syntax but there is no easy way to write it directly from the input.

![Screenshot 2022-10-12 at 12 17 05](https://user-images.githubusercontent.com/19715763/195317550-796cbf90-f617-49ba-a389-54ce86a464b4.png)

Could you please add the Rich Text Editor there?

Thanks in advance.

**Comment by sarmadgardezi** [comment]
Have you tried the Remote-Mdx ?
